README.md

```diff
@@ -48,14 +47,6 @@ This project is a table where you cand find animals and their features
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Kindly fix the following few issues with the readme file

1. Clicking on the `Deployment` link in the table of contents should take the user to the deployment part of the `Getting Started` section, kindly fix this broken link.
![image](https://github.com/Felipe-Perez-Ferraro/vet-clinic/assets/81932478/816df999-ae9e-4e3e-9a34-8463bc56857e)

2. Kindly fix the broken link to the issues page under the `Contributing` section for a more professional readme file.
![image](https://github.com/Felipe-Perez-Ferraro/vet-clinic/assets/81932478/585d7cbf-56f9-4d88-8057-c0da61e83503)